astropy/io/ascii/tests/test_write.py

```diff
@@ -15,21 +15,22 @@
 from .common import setup_function, teardown_function
 
 test_defs = [
-    dict(kwargs=dict(),
+    dict(kwargs=dict(Writer=ascii.Basic),
```

Comment:
Why do you specify the writer here? The intentions of this test is not only to test the basic writer, but also to make sure that the default option (which currently is the Basic writer) always gives the same answer. Most users actually don't set the Writer by hand, they just rely on the default, so we should test it that way, too.